test/unit/ngsi20/examples/contextRequests/updateContext1.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Directory name change: ngsi20/ -> ngsi2/ (or maybe even better ngsiv2)

Note that although NGSI10 had some meaning (although it is not a good idea to keep it, as I explain in another comment), NGSI20 doesn't have a meaning.

Reply:
Done